product_mandatory_products/views/template.xml

```diff
@@ -0,0 +1,376 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <template id="mandatory_products_modal" name="Optional Products">
+        <main class="modal-body">
+            <t t-call="product_mandatory_products.configure_mandatory_products" />
+        </main>
+    </template>
+
+    <template id="product_quantity_config">
+        <div class="css_quantity input-group">
+            <div class="input-group-prepend">
+                <button
+          t-attf-href="#"
+          class="btn btn-primary float_left js_add_cart_json d-none d-md-inline-block"
+          aria-label="Remove one"
+          title="Remove one"
+        >
+                    <i class="fa fa-minus" />
+                </button>
+            </div>
+            <input
+        type="text"
+        class="js_quantity form-control quantity"
+        data-min="1"
+        name="add_qty"
+        t-att-value="add_qty or 1"
+      />
+            <div class="input-group-append">
+                <button
+          t-attf-href="#"
+          class="btn btn-primary float_left js_add_cart_json d-none d-md-inline-block"
+          aria-label="Add one"
+          title="Add one"
+        >
+                    <i class="fa fa-plus" />
+                </button>
+            </div>
+        </div>
+    </template>
```

Comment:
Formatting looks partially very weird.

Reply:
@manengel formatted the code.
@jans23 Shall I remove the commented code or I ask nikunj related to it?

Reply:
Feel free to remove commented code.

Reply:
@jans23 I've removed the extra commented code.